content/applications/sales/sales/products_prices/returns/returns_refunds.rst

```diff
@@ -0,0 +1,104 @@
+customers.
+
+The Sales App provides two different ways to process returns based on whether an invoice has been
+sent or not.
+
+Case 1: Using Reverse Transfers
+===============================
+When a customer decides to return a product before an invoice has been sent or validated,
+returns are completed using reverse transfers.
+
+To start a return, navigate to the customer’s Sales Order, and click on the Delivery smart tab
+to open the associated Delivery Order. On the validated Delivery Order, the available buttons are
+**Print**, **Return**, and **Unlock**.
+
+Click **Return**, which opens the Reverse Transfer pop-up. By default, the quantities match the
+validated quantities from the Delivery Order. Update the quantities if the customer wants to
+return a portion of the validated amount. Clicking the trash icon next to a line item removes the
+item without having to zero out the quantity.
+
+.. image:: returns_refunds/Reverse-transfer-popup.png
+  :align: center
+  :alt: Reverse Transfer Pop-Up
+
+Clicking **Return** generates a new Warehouse Operation for the incoming returned product(s). After
+the receiving the return, the team can validate the Warehouse Operation. Head back to the original
+Sales Order, and the Delivered Quantity reflects the difference between the original and the
+returned quantities.
+
+.. image:: returns_refunds/updated-sales-quantities.png
+  :align: center
+  :alt: Updated Sales Order Quantities
+
+The customer receives an invoice only for the products they're keeping.
+
+Case 2: Returning products after invoicing
+==========================================
+Sometimes, customers receive and/or pay their invoice before deciding to return an item within
+your company’s return window. In these cases, a return using only Reverse Transfers is impossible
+since validated or sent invoice cannot be changed, but Reverse Transfers can be used in
+conjunction with Credit Notes to complete the customer's return.
+
+Navigate to the customer’s Sales Order. If a Sales Order has any registered payment on it, the
+payment details are located in the chatter, and the invoice itself has a **Paid** banner across it.
+
+A customer wants to return 10 of the 20 products that they purchased, already
+paid for, and received. Starting a return in this case begins the same as Case 1 by clicking on
+the validated Delivery Order, then on  **Return** to open the Reverse Transfer pop-up. Edit the
+quantity from 20 to 10 since the customer only wants to return half of the order, and click
+**Return**. Just as in Case 1, this action generates a new Warehouse Operation for the incoming
+returned product, which can then be validated once received. Back on the Sales Order, if **Update
+Quantities** was selected, the return product's line item now reflects the change in amount under
+delivered quantities.
+
+.. image:: returns_refunds/case-2-updated-sales-quantities.png
+  :align: center
+  :alt: Updated Delivered Quantities
```

Comment:
The screenshot is huge. Check my previous comments on screenshots to improve that one as well